app/views/books/_books_search.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= form_tag(books_path, method: :get, id: 'search_form') do %>
+  <%= text_field_tag :search, params[:search], placeholder: "Search by book or by author" %>
```

Comment:
You may want to make the placeholder text use i18n so it matches the rest of the app